pmm-app/src/pmm-inventory/panel.test.tsx

```diff
@@ -56,18 +56,16 @@ describe('Inventory tables', () => {
     act(() => {
       render(
         <Table
-          dataSource={InventoryDataService.generateStructure(response)}
+          data={InventoryDataService.generateStructure(response)}
           rowKey={rec => rec.agent_id}
-          columns={agentsColumns}
-          pagination={false}
-          bordered
+          columns={AGENTS_COLUMNS}
           loading={false}
         />,
         container
       );
     });
     // length is 5 because header is also tr
-    expect(container?.querySelectorAll('tr').length).toEqual(5);
+    expect(container.querySelectorAll('tr').length).toEqual(5);
```

Comment:
Pls use enzyme `shallow` or `mount` to make it consistent with the rest of the app.